ui/tests/e2e/1000-genomes.test.js

```diff
@@ -164,6 +164,23 @@ describe("End-to-end 1000 genomes", () => {
     await waitForFacetsUpdate(477);
   });
 
+  test("Facet with range intervals", async () => {
```

Comment:
I was thinking a unit test instead of e2e for this bug. I looked at unit tests and I think they're good enough. Let's revert this file and not add any new tests in this PR. Thanks.

Reply:
Reverted this file. Thanks.